src/game/localization.cpp

```diff
@@ -42,7 +41,7 @@ void CLocalizationDatabase::AddString(const char *pOrgStr, const char *pNewStr,
 	s.m_Hash = str_quickhash(pOrgStr);
 	s.m_ContextHash = str_quickhash(pContext);
 	s.m_pReplacement = m_StringsHeap.StoreString(*pNewStr ? pNewStr : pOrgStr);
-	m_Strings.add(s);
+	m_Strings.push_back(s);
```

Comment:
This should probably be:
```suggestion
	m_Strings.emplace_back(CString(str_quickhash(pOrgStr), str_quickhash(pContext), m_StringsHeap.StoreString(*pNewStr ? pNewStr : pOrgStr));
```

Avoiding an extra copy

Reply:
I think you placed `CString(` wrong there?! @Learath2